pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -406,6 +406,9 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1.PipelineRun, getPipel
 		message := fmt.Sprintf("PipelineRun %s/%s awaiting remote resource", pr.Namespace, pr.Name)
 		pr.Status.MarkRunning(ReasonResolvingPipelineRef, message)
 		return nil
+	case errors.Is(err, resources.ErrRemotePipelineValidationFailed):
+		pr.Status.MarkFailed(ReasonCouldntGetPipeline, err.Error())
```

Comment:
Wondering if we want to mark the status as `ReasonFailedValidation` instead of `ReasonCouldntGetPipeline` because it was a validation failure?

Reply:
This is a good question, open to alternatives. I'm using `ReasonCouldntGetPipeline` because that's what we currently use when there's an error with pipeline resolution: https://github.com/tektoncd/pipeline/blob/d2cb90de128c85d1e3915fae21c0601c0a22845f/pkg/reconciler/pipelinerun/pipelinerun.go#L410-L415
ReasonFailedValidation or ReasonFailedRemoteValidation are also reasonable choices. This could make sense to address as part of a more thorough deep dive into our error messages and reasons? Curious to hear your thoughts.

Reply:
Thanks for the context of #6859, that clarifies the ways forward here. Would it make sense to include the issue in the docString or comment in the issue just for reminders?

Reply:
Fair enough; done.